main_logic/tts_client.py

```
@@ -636,17 +642,31 @@ async def receive_messages_initial():
+                        try:
+                            await ws.close()
+                        except Exception:
+                            pass
+                        ws = None
+                    session_ready.clear()
+
             receive_task = asyncio.create_task(receive_messages_initial())
             
             # 主循环：处理请求队列
             loop = asyncio.get_running_loop()
+            pending = None  # 断线重试时暂存当前片段，保证顺序（不回共享队列）
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)